packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/NavigationDrawerItem.tsx

```diff
@@ -268,13 +271,37 @@ export const NavigationDrawerItem = ({
       onClick();
       return;
     }
+
+    if (
```

Comment:
code seems duplicated

Reply:
You mean we should introduce a utiler for event.metaKey && ... ? I did initially then removed it because my name wasn't great. I can bring to find a better one and bring it back